CMSgov/mint-app
# Discussion Tagging Feature

# EASI-3287

## Changes and Description

- Front End Changes
   - [Frontend PR](https://github.com/CMSgov/mint-app/pull/779)
- Backend Changes
     - [Backend PR](https://github.com/CMSgov/mint-app/pull/778)
    - [email PR ](https://github.com/CMSgov/mint-app/pull/802)



## How to test this change
1. Create tags in a plan discussion. Verify tagging functions as anticipated from the previous PRS

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
